### PR TITLE
xdm: new cask, 7.2.7

### PR DESCRIPTION
**Pull request by yurikoles**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/xdm.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  appcast 'https://sourceforge.net/projects/xdman/rss'
+  name 'Xtreme Download Manager'
+  homepage 'http://xdman.sourceforge.net/'
+
+  installer script: {
+                      executable: "#{staged_path}/install",
+                      sudo:       true,
+                    }
+
+  uninstall delete: [
```
Comment (vitorgalvao) [inline Casks/xdm.rb]: It’s unlikely this is all the script does. Isn’t there an `uninstaller`?

Also, please pay attention to the corrections. You’ve been submitting for years, probably, and still make basic mistakes like not separating the array properly.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/xdm.rb (hunk 1 above)
Comment (yurikoles) [reply Casks/xdm.rb]: from [official ticket](https://sourceforge.net/p/xdman/discussion/xdmhelp/thread/701f5570/):

>* delete the xdm.app folder from applications
>* open terminal, and type rm -rf ~/.xdman

Comment (yurikoles) [reply Casks/xdm.rb]: @vitorgalvao we have ticket about arrays: https://github.com/Homebrew/rubocop-cask/issues/64

Comment (yurikoles) [reply Casks/xdm.rb]: And, no, I can't find any uninstaller in dmg or app dir.

Comment (vitorgalvao) [reply Casks/xdm.rb]: > we have ticket about arrays: Homebrew/rubocop-cask#64

Yes, but that’s about `style` doing it automatically, nothing stops you from doing it manually, which is what everyone else does.

File: Casks/xdm.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                      '/Applications/xdm.app/',
```
Comment (vitorgalvao) [inline Casks/xdm.rb]: Remove the trailing slash.

File: Casks/xdm.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+                      '~/Library/LaunchAgents/org.sdg.xdman.plist',
+                      '~/Desktop/xdm.app',
```
Comment (vitorgalvao) [inline Casks/xdm.rb]: It installs one app in `/Applications` *and* another one with the same name in the `Desktop`?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/xdm.rb (hunk 3 above)
Comment (yurikoles) [reply Casks/xdm.rb]: I no longer see app on Desktop after fresh install, seems to be issue in my setup.

**Review by victorpopkov — APPROVED** [review]

**Pull request merged**